backend-agent/app/db/utils.py

```diff
@@ -0,0 +1,71 @@
+from .models import (
+    Attack as AttackDB,
+    db,
+    AttackModel as AttackModelDB,
+    AttackResult as AttackResultDB,
+    ModelAttackScore as ModelAttackScoreDB,
+)
+
+
+def save_to_db(attack_results):
+    """
+    Persist the SuiteResult into the database.
```

Comment:
only the SuiteResult or any attack (also run via the CLI instead of using the agent)?